packages/notifier-seeder/src/index.ts

```diff
@@ -58,34 +68,53 @@ export function processMessage(
         const message = buildPurposeMessage(decodedMessage, event);
         return { decodedMessage, message };
       })
+      .with(authorizationTopic.authorizationTopic, () => {
+        const decodedMessage = decodeKafkaMessage(
+          kafkaMessage.message,
+          AuthorizationEventV2
+        );
+
+        const event = toAuthorizationEventNotification(decodedMessage);
+
+        const message = event
+          ? buildAuthorizationMessage(decodedMessage, event)
+          : undefined;
+        return { decodedMessage, message };
+      })
       .otherwise(() => {
         throw new Error(`Unknown topic: ${kafkaMessage.topic}`);
       });
 
-    const loggerInstance = logger({
-      serviceName: "notifier-seeder",
-      eventType: decodedMessage.type,
-      eventVersion: decodedMessage.event_version,
-      streamId: decodedMessage.stream_id,
-      correlationId: decodedMessage.correlation_id,
-    });
-    if (decodedMessage.event_version !== 2) {
+    if (message) {
+      const loggerInstance = logger({
+        serviceName: "notifier-seeder",
+        eventType: decodedMessage.type,
+        eventVersion: decodedMessage.event_version,
+        streamId: decodedMessage.stream_id,
+        correlationId: decodedMessage.correlation_id,
+      });
+      if (decodedMessage.event_version !== 2) {
+        loggerInstance.info(
+          `Event with version ${decodedMessage.event_version} skipped`
+        );
+        return;
+      }
+
+      await queueManager.send(message, loggerInstance);
+
       loggerInstance.info(
-        `Event with version ${decodedMessage.event_version} skipped`
+        `Notification message [${message.messageUUID}] sent to queue ${queueConfig.queueUrl} for event type "${decodedMessage.type}"`
```

Comment:
Minor: The `decodedMessage.type` is already present in the `loggerInstance` metadata